src/network/networkprotocol.h

```diff
@@ -538,20 +539,25 @@ enum ToClientCommand
 
 	TOCLIENT_ADD_PARTICLESPAWNER = 0x47,
 	/*
-		-- struct range<T> { T min, T max, f32 bias };
-		-- struct tween<T> { T start, T end };
+		-- using range<T> = RangedParameter<T>;
```

Comment:
```suggestion
		-- using range<T> = RangedParameter<T>
		   {
		       u8 style
		       u16 reps
		       f32 beginning
		       T start
		       T end
		   }
```

Reply:
Well sure whatever. Although I don't really see the need to again define the exact format here when it's already contained in its dedicated struct/class.